src/burp/restore.epp

```diff
@@ -3057,7 +3059,10 @@ rec_type get_data(BurpGlobals* tdgbl, burp_rel* relation, bool skip_relation)
 			}
 			else
 			{
-				if (tdgbl->gbl_sw_incremental)
+				if (tdgbl->gbl_sw_incremental &&
```

Comment:
I agree with an idea to make error processing during restore more flexible. But list of fatal errors appears to be incomplete. Except isc_net_write_err isc_net_read_err can happen, except isc_att_shutdown - isc_shutdown, etc. On the other hand all such gds codes are already classified as SQL code -902. Why not use it instead?